.github/workflows/ci.yml

```diff
@@ -28,23 +28,27 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9, "3.10", "3.11", "3.12"]
+        python-version: [3.9, "3.10", "3.11", "3.12", "3.13"]
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Install poetry
         run: pipx install poetry
 
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
           python-version: "${{ matrix.python-version }}"
           cache: "poetry"
 
       - name: Install dependencies
         run: poetry install
 
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: "1.10.1"
```

Comment:
hmm.. I forget if we need to not install the typescript wrapper the setup-tf action uses. but tests seem to pass okay.